File: 05_이원영/week4/11399.js
//15-5 11399
//var input = require('fs').readFileSync('/dev/stdin'); //임시 입력 처리

const N = 5;
var arr = [3, 1, 4, 3, 2];
var time = 0;

arr.sort(); //각 시간은 가중되기 때문에 그리디 알고리즘을 바탕으로 가장 작은 수부터 우선 배치

var add = 0;
for (var i = 0; i<N;i++) { //마지막 사람이 기다리는 총 시간
    add += arr[i];
    time += add;
}

console.log(time);

File: 09_주정윤/week4/11399.py
# 사람의 수
N = int(input())

time = list(map(int, input().split()))

time.sort()
sum=0
sum2=0
sum_list=[]

for i in range(N):
    sum = sum + time[i] 
    sum2 = sum2+sum
 

print(sum2)
